core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraXCameraUseCase.kt

```diff
@@ -248,7 +250,7 @@ constructor(
         currentSettings.value =
             cameraAppSettings
                 .tryApplyDynamicRangeConstraints()
-                .tryApplyAspectRatioForExternalCapture(externalImageCapture)
+                .tryApplyAspectRatioForExternalCapture(disableVideoCapture)
```

Comment:
We should also disable image capture when we're in external intent video capture mode, and handle it accordingly here. Maybe the `disableVideoCapture` boolean should be changed to a `BitSet` of use cases that are enabled / disabled.

Also, the aspect ratio in this method should be set to `SIXTEEN_NINE` explicitly in that case.

Reply:
I tried that but not attaching imageCapture use case causes other problems that increases the complexity of this class by a lot.

Reply:
Instead of BitSet I used an enum with STANDARD, VIDEO_ONLY, and IMAGE_ONLY, since the three modes are mutually exclusive. If we introduce more variants in the future, we could consider using bitset